### DISCUSSION TITLE
Get a `bruno` tag created on Stack Overflow

### DISCUSSION THREAD

**Discussion by Andreas-Schoenefeldt** [usebruno/bruno#865] [General]
In order to have an additional support channel next to github issues, it might be good to have the `bruno` tag created in stack overflow.

Unfortunately I don't have enough reputation to do it myself, but maybe somebody else here has?

**Comment by koriit-kontakt** [comment]
>  Creation privilege awarded at 1,500 reputation

> on some sites, new tags will be automatically culled and removed from the system if they are not used by at least 1 other question in a 6 month period.

https://stackoverflow.com/help/privileges/create-tags

**Reply by Andreas-Schoenefeldt** [reply]
Actually I have more the 1.5k - I'll try that.

**Comment by Andreas-Schoenefeldt** [comment]
We have it :D - https://stackoverflow.com/tags/bruno Worked indeed better then I thought :D - I'll add a PR for some addition in the readme

**Comment by Andreas-Schoenefeldt** [comment]
Here is the PR for changing the readme: https://github.com/usebruno/bruno/pull/926/files

Please feel free to comment / edit there as you seem fit.
